doc/man3/EVP_MAC.pod

```diff
@@ -279,14 +279,16 @@ This option is used by KMAC.
 A simple flag to set the MAC digest to not initialise the
 implementation specific data. The value 0 or 1 is expected.
 
-This option is used by HMAC.
+This option is deprecated and will be removed in a future release.
+The option may be set, but is ignored.
 
 =item "digest-oneshot" (B<OSSL_MAC_PARAM_DIGEST_ONESHOT>) <integer>
 
 A simple flag to set the MAC digest to be a oneshot operation.
 The value 0 or 1 is expected.
 
-This option is used by HMAC.
+This option is deprecated and will be removed in a future release.
+The option may be set, but is ignored.
```

Comment:
Is there a way to validate a config file, specifying whether deprecated options are allowed or not?

People end up with all sort of cruft in these files, they build them up from what they find on the Internet (even though they shouldn't) and a way of checking that the config makes sense and isn't using deprecated options could well be useful

Reply:
I am not sure what kind of config file you have on mind here. These are OSSL_PARAMS possible with the EVP_MAC implementations. Anything unknown is ignored.

Reply:
Ah, okay, nevermind then. Sorry for the noise